kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/canvas/controls/BPMNInlineTextEditorControlTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.workbench.common.stunner.bpmn.client.canvas.controls;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kie.workbench.common.stunner.bpmn.definition.BaseCatchingIntermediateEvent;
+import org.kie.workbench.common.stunner.bpmn.definition.DataObject;
+import org.kie.workbench.common.stunner.bpmn.definition.EmbeddedSubprocess;
+import org.kie.workbench.common.stunner.bpmn.definition.EndCompensationEvent;
+import org.kie.workbench.common.stunner.bpmn.definition.EventGateway;
+import org.kie.workbench.common.stunner.bpmn.definition.StartCompensationEvent;
+import org.kie.workbench.common.stunner.bpmn.definition.TextAnnotation;
+import org.kie.workbench.common.stunner.bpmn.definition.UserTask;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doCallRealMethod;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BPMNInlineTextEditorControlTest {
+
+    @Mock
+    private BPMNInlineTextEditorControl inline;
+
+    @Mock
+    private EventGateway gateway;
+
+    @Mock
+    private EndCompensationEvent endEvent;
+
+    @Mock
+    private StartCompensationEvent startEvent;
+
+    @Mock
+    private BaseCatchingIntermediateEvent intermediateEvent;
+
+    @Mock
+    private UserTask userTask;
+
+    @Mock
+    private EmbeddedSubprocess embeddedSubprocess;
+
+    @Mock
+    private DataObject dataObject;
+
+    @Mock
+    private TextAnnotation textAnnotation;
+
+    @Test
+    public void testEditable() {
```

Comment:
Again you are testing multiple units it one test. If for example `assertEquals(true, inline.isFiltered(userTask));` fails others asserts will not be run.

Reply:
Hi @domhanak split now into two tests.

Reply:
I do not see this being split.

Reply:
Please test each node separately